src/app/page.tsx

```diff
@@ -45,7 +45,7 @@ export default function Home() {
                 // fill
                 style={{ justifyContent: 'center' }}
               />
-              <Title align="left" color="red" fw={700} size={80}>
+              <Title align="left" color="red.8" fw={700} size={80}>
```

Comment:
interesting '.8'

Reply:
Thought it would be better to just use Mantine's default color for better consistency.
https://mantine.dev/theming/colors/